app/controllers/ops_controller/settings/common.rb

```diff
@@ -233,6 +241,43 @@ def pglogical_validate_subscription
     javascript_flash
   end
 
+
+  def enable_central_admin
+    replication_type = MiqRegion.replication_type
+
+    if replication_type != :global || !@_params[:provider_region] || !@_params[:ssh_user] || !@_params[:ssh_password]
+      add_flash(_("Invalid data for enabling Central Admin"), :error)
+    else
+      provider_region = @_params[:provider_region]
+      region = MiqRegion.where(:region => provider_region).first
+      if region
+        region.generate_auth_key_queue(@_params[:ssh_user], @_params[:ssh_password], @_params[:ssh_host])
+        add_flash(_("Enable Central Admin has been successfully initiated"))
+      else
+        add_flash(_("Region Not found"), :error)
+      end
+    end
+    javascript_flash
+  end
+
+  def disable_central_admin
+    replication_type = MiqRegion.replication_type
+
+    if replication_type != :global || !@_params[:provider_region]
+      add_flash(_("Invalid data for disabling Central Admin"), :error)
+    else
+      provider_region = @_params[:provider_region]
+      region = MiqRegion.where(:region => provider_region).first
```

Comment:
Same here also.
`MiqRegion.find_by_region(provider_region)`
I thought there was a rubocop cop for `where().first` ...